### DISCUSSION TITLE
Network Issues During Installation

### DISCUSSION THREAD

**Discussion by andronicus3** [Security-Onion-Solutions/securityonion#2567] [Unsupported Versions] [answered]
Hi

I am having a network issue during the securityonion network configuration. My network adapters have two distinct IP addresses on the same subnet. 
 
"The IP being routed by Linux is not the IP address assigned to the management interface (enp0s3)

This is not a supported configuration, please remediate and rerun."

My current 2 network adapters have x.x.x.55 (static IP) and x.x.x.140 (dhcp).

Should both adapters have the same IP address? What is the best way to change either IP address? The installer menu doesn't allow you to move back to a previous screen.

Please let me know. Thanks.

**Comment by TOoSmOotH** [comment]
I am assuming this is a network install? What OS are you using? Assuming this is a network install you would be better off disabling the monitor interface then run the install. The install will make the appropriate changes to make it work.

**Comment by andronicus3** [comment, marked as answer]
Thanks. I got it running with the terminal-only install. I am using separate IP addresses for each interface. This is a Virtualbox install on Windows 10. Thanks again.
